service/src/test/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredTaskExecutorTest.java

```diff
@@ -86,6 +89,54 @@ public void When_BundleContainsExternalDocumentReference_Expect_DocumentAddedToE
         );
     }
 
+    @ParameterizedTest
+    @EmptySource
```

Comment:
Apparently this `EmptySource` is how you can add NULL to a parameterized test @MartinWheelerMT .

Reply:
Good to know, I thought it would be possible

Reply:
Actually I think this is wrong, it looks like @NullSource is for nulls, @EmptySource for empty strings, @NullAndEmptySource for both:

@EmptySource is an [ArgumentsSource](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/params/provider/ArgumentsSource.html) which provides a single empty argument to the annotated @ParameterizedTest method.

@NullSource is an [ArgumentsSource](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/params/provider/ArgumentsSource.html) which provides a single null argument to the annotated @ParameterizedTest method.

Or combine the two:
@NullAndEmptySource is a composed annotation that combines the functionality of [@NullSource](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/params/provider/NullSource.html) and [@EmptySource](https://junit.org/junit5/docs/5.4.1/api/org/junit/jupiter/params/provider/EmptySource.html).

Reply:
For reason it lowercases `@NullSource`